components/Plate/Editor.jsx

```diff
@@ -386,9 +279,10 @@ export default function Editor({
     imageUrlGetter
   } = useImageUrlGetterAndSaveCallback()
 
-  // use the standard combobox because we're using the mentions stuff
-  const mentionItems = useComboboxItems(P.comboboxStore, mentionNames)
-  const tagItems = useComboboxItems(P.comboboxStore, tagNames)
+  // as of 2/14/22 we aren't passing mentions or tags into Editor, so these don't
+  // do anything. We may want to change this soon, so I'll leave this here for now - TV
```

Comment:
Added a ticket for tags, I think we should be able to do that pretty easily soon.